roomserver/alias/alias_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2017 Vector Creations Ltd
```

Comment:
Can you update this to the current year and your name please? :)

Reply:
Whoops, new to contributing like this!